src/rpc/mod.rs

```diff
@@ -284,3 +289,7 @@ where
 
     Ok(())
 }
+
+fn map_rpc_err(e: impl Display) -> RpcError {
```

Comment:
nit: maybe rename function to `to_rpc_err`?

Reply:
Fixed.